### DISCUSSION TITLE
Request to add a repeat() function... please...

### DISCUSSION THREAD

**Discussion by Minamotion** [replit/kaboom#821] [Ideas]
### Concept
It can work like 
```javascript
repeat(5, ()=>{  /*kaboom code goes here*/  })
``` 
what it does is repeat 5 times some code, it can work with `await`
### Why I want this
I was working on a dialog system and found the inevitable bug of `loop()` making the text delay be subtracted each dialog line, this is how i used it in a dialog class:
```javascript
processDialog(dialog = ["Hello World!", "I want burgur", "burgur", "burgur pls"], delay = 0.01) {
        if (!(this.dialogfinished)) {
            this.text = ""
            if (dialog.length > 0) {
                this.text = ""
                this.dtext = dialog.shift().split("")
                loop(delay, ()=>{
                    if(this.dtext.length > 0){
                        this.text += this.dtext.shift()
                    }
                })
            } else {
                this.dialogfinished = true
                this.msgbox.destroy()
            }
        }
    }
```

**Comment by Minamotion** [comment]
Update: Nvm, I found a way to fix my issue LOL, but yet it will still be useful

**Reply by Minamotion** [reply]
[**I even made an entire github repository about this xd, just click this text and you'll be redirected to it**](https://github.com/Minamotion/kaboomjs-dialog-system)

**Comment by slmjkdbtl** [comment]
Does loop accompolish what you want to do?
```ts
for (let i = 0; i < 5; i++) {
    // do whatever
}
```

**Reply by Minamotion** [reply]
Wait, you're right!

**Reply by Minamotion** [reply]
I could do that, I'm dumb

**Reply by Minamotion** [reply]
Whatever, thanks man :)
